user-info-provider/src/main/java/de/intension/mapper/oidc/UserInfoProviderMapper.java

```diff
@@ -0,0 +1,186 @@
+
+    private static final UserInfoHelper               userInfoHelper           = new UserInfoHelper();
+
+    static {
+        addConfigEntry(HEIMATORGANISATION_NAME);
+        addConfigEntry(HEIMATORGANISATION_BUNDESLAND);
+        addConfigEntry(PERSON_FAMILIENNAME);
+        addConfigEntry(PERSON_FAMILIENNAME_INITIALEN);
+        addConfigEntry(PERSON_VORNAME);
+        addConfigEntry(PERSON_VORNAME_INITIALEN);
+        addConfigEntry(PERSON_AKRONYM);
+        addConfigEntry(PERSON_GEBURTSDATUM);
+        addConfigEntry(PERSON_ALTER);
+        addConfigEntry(PERSON_GESCHLECHT);
+        addConfigEntry(PERSON_LOKALISIERUNG);
+        addConfigEntry(PERSON_VERTRAUENSSTUFE);
+        addConfigEntry(PERSON_KONTEXT_ORG_VIDIS_ID);
+        addConfigEntry(PERSON_KONTEXT_ORG_KENNUNG);
+        addConfigEntry(PERSON_KONTEXT_ORG_NAME);
+        addConfigEntry(PERSON_KONTEXT_ORG_TYP);
+        addConfigEntry(PERSON_KONTEXT_ROLLE);
+        addConfigEntry(PERSON_KONTEXT_STATUS);
+        addOnlyChildrenClaims();
+        OIDCAttributeMapperHelper.addAttributeConfig(configProperties, UserInfoProviderMapper.class);
+        setDefaultTokenClaimNameValue();
+        setDefaultTokenClaimType();
+        deactivateAccessToken();
+    }
+
+    /**
+     * Set default for field "claim.name".
+     */
+    private static void setDefaultTokenClaimNameValue()
+    {
+        Optional<ProviderConfigProperty> config = configProperties.stream().filter(p -> p.getName().equals(OIDCAttributeMapperHelper.TOKEN_CLAIM_NAME))
+            .findFirst();
+        if (!config.isEmpty()) {
+            config.get().setDefaultValue(USER_INFO_ATTRIBUTE_NAME);
+        }
+    }
+
+    /**
+     * Set default for field "jsonType.label".
+     */
+    private static void setDefaultTokenClaimType()
+    {
+        Optional<ProviderConfigProperty> config = configProperties.stream().filter(p -> p.getName().equals(OIDCAttributeMapperHelper.JSON_TYPE))
+            .findFirst();
+        if (!config.isEmpty()) {
+            config.get().setDefaultValue("JSON");
+        }
+    }
+
+    /**
+     * Deactivate ACCESS_TOKEN as a default target storing users metadata.
+     */
+    private static void deactivateAccessToken()
+    {
+        Optional<ProviderConfigProperty> config = configProperties.stream().filter(p -> p.getName().equals(OIDCAttributeMapperHelper.INCLUDE_IN_ACCESS_TOKEN))
+            .findFirst();
+        if (!config.isEmpty()) {
+            config.get().setDefaultValue("false");
+        }
+    }
+
+    /**
+     * Add custom configuration entries.
+     */
+    private static void addConfigEntry(UserInfoAttribute attribute)
+    {
+        ProviderConfigProperty property = new ProviderConfigProperty();
+        property.setName(attribute.getAttributeName());
+        property.setLabel(attribute.getLabel());
+        property.setDefaultValue(attribute.isEnabled().toString());
+        property.setType(ProviderConfigProperty.BOOLEAN_TYPE);
```

Comment:
Can you add a help text explaining what they're for? :thinking: 
![image](https://user-images.githubusercontent.com/42644709/207825373-7aaa42e5-957d-4eab-b01c-0a711cddd702.png)

Reply:
Sure

Reply:
No localization - but fine for now hm :+1: